src/cmd/nsfs.js

```diff
@@ -244,6 +244,7 @@ class NsfsAccountSDK extends AccountSDK {
 async function main(argv = minimist(process.argv.slice(2))) {
     try {
         config.DB_TYPE = 'none';
+        config.PROMETHEUS_ENABLED = false;
```

Comment:
Can you explain this change? we want to start the prometheus metrics on NC

Reply:
if we will disable it we won't start the metrics server on NC, check the code of start_server(), specifically when forks_enabled - which only true when using NC currently

Reply:
you can try test it by running `noobaa-cli diagnose metrics` command

Reply:
Thank you, by disabling and running `sudo node src/cmd/manage_nsfs diagnose metrics` I see:

> Nov-28 11:06:47.257 [Upgrade/41904]  [WARN] core.manage_nsfs.diagnose:: could not receive metrics response AggregateError [ECONNREFUSED]:
>     at internalConnectMultiple (node:net:1118:18)
>     at afterConnectMultiple (node:net:1685:7) {
>   code: 'ECONNREFUSED',
>   [errors]: [
>     Error: connect ECONNREFUSED ::1:7004
>         at createConnectionError (node:net:1648:14)
>         at afterConnectMultiple (node:net:1678:16) {
>       errno: -61,
>       code: 'ECONNREFUSED',
>       syscall: 'connect',
>       address: '::1',
>       port: 7004
>     },
>     Error: connect ECONNREFUSED 127.0.0.1:7004
>         at createConnectionError (node:net:1648:14)
>         at afterConnectMultiple (node:net:1678:16) {
>       errno: -61,
>       code: 'ECONNREFUSED',
>       syscall: 'connect',
>       address: '127.0.0.1',
>       port: 7004
>     }
>   ]
> }
> {
>   "error": {
>     "code": "MetricsStatusFailed",
>     "message": "Metrics fetch request failed",
>     "cause": "Error: connect ECONNREFUSED ::1:7004\n    at createConnectionError (node:net:1648:14)\n    at afterConnectMultiple (node:net:1678:16)"
>   }
> }

unlike something like:

> {
>   "response": {
>     "code": "MetricsStatus",
>     "reply": {
>       "nsfs_counters": {
>         "noobaa_nsfs_io_read_count": 1,
>         "noobaa_nsfs_io_write_count": 0,
>         "noobaa_nsfs_io_read_bytes": 268435456,
>         "noobaa_nsfs_io_write_bytes": 0
>       },
>       "op_stats_counters": {
>         "noobaa_nsfs_op_head_object_min_time_milisec": 2,
>         "noobaa_nsfs_op_head_object_max_time_milisec": 2,
>         "noobaa_nsfs_op_head_object_avg_time_milisec": 2,
>         "noobaa_nsfs_op_head_object_count": 1,
>         "noobaa_nsfs_op_head_object_error_count": 0,
>         "noobaa_nsfs_op_read_object_min_time_milisec": 367,
>         "noobaa_nsfs_op_read_object_max_time_milisec": 367,
>         "noobaa_nsfs_op_read_object_avg_time_milisec": 367,
>         "noobaa_nsfs_op_read_object_count": 1,
>         "noobaa_nsfs_op_read_object_error_count": 0
>       },
>       "fs_worker_stats_counters": {
>         "noobaa_nsfs_fs_worker_readfile_min_time_milisec": 61,
>         "noobaa_nsfs_fs_worker_readfile_max_time_milisec": 266,
>         "noobaa_nsfs_fs_worker_readfile_avg_time_milisec": 182,
>         "noobaa_nsfs_fs_worker_readfile_count": 3,
>         "noobaa_nsfs_fs_worker_readfile_error_count": 0,
>         "noobaa_nsfs_fs_worker_stat_min_time_milisec": 33,
>         "noobaa_nsfs_fs_worker_stat_max_time_milisec": 416,
>         "noobaa_nsfs_fs_worker_stat_avg_time_milisec": 85,
>         "noobaa_nsfs_fs_worker_stat_count": 11,
>         "noobaa_nsfs_fs_worker_stat_error_count": 0,
>         "noobaa_nsfs_fs_worker_fileopen_min_time_milisec": 25,
>         "noobaa_nsfs_fs_worker_fileopen_max_time_milisec": 25,
>         "noobaa_nsfs_fs_worker_fileopen_avg_time_milisec": 25,
>         "noobaa_nsfs_fs_worker_fileopen_count": 1,
>         "noobaa_nsfs_fs_worker_fileopen_error_count": 0,
>         "noobaa_nsfs_fs_worker_filestat_min_time_milisec": 66,
>         "noobaa_nsfs_fs_worker_filestat_max_time_milisec": 66,
>         "noobaa_nsfs_fs_worker_filestat_avg_time_milisec": 66,
>         "noobaa_nsfs_fs_worker_filestat_count": 1,
>         "noobaa_nsfs_fs_worker_filestat_error_count": 0,
>         "noobaa_nsfs_fs_worker_fileread_min_time_milisec": 4,
>         "noobaa_nsfs_fs_worker_fileread_max_time_milisec": 5127,
>         "noobaa_nsfs_fs_worker_fileread_avg_time_milisec": 1907,
>         "noobaa_nsfs_fs_worker_fileread_count": 33,
>         "noobaa_nsfs_fs_worker_fileread_error_count": 0,
>         "noobaa_nsfs_fs_worker_fileclose_min_time_milisec": 20,
>         "noobaa_nsfs_fs_worker_fileclose_max_time_milisec": 20,
>         "noobaa_nsfs_fs_worker_fileclose_avg_time_milisec": 20,
>         "noobaa_nsfs_fs_worker_fileclose_count": 1,
>         "noobaa_nsfs_fs_worker_fileclose_error_count": 0
>       }
>     }
>   }
> }

So I won't disable it on NC.